plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -89,6 +89,9 @@ fu_plugin_thunderbolt_composite_cleanup(FuPlugin *plugin, GPtrArray *devices, GE
 		FuDevice *dev = g_ptr_array_index(devices, i);
 		if ((g_strcmp0(fu_device_get_plugin(dev), "thunderbolt") == 0) &&
 		    fu_device_has_internal_flag(dev, FU_DEVICE_INTERNAL_FLAG_NO_AUTO_REMOVE)) {
+			g_usleep(
```

Comment:
Since this is a value that you may tune later - maybe put it in `#define` at the top and include your comment there.
then this can just be something like: 
`g_usleep(RETIMER_REPLUG_DELAY);`

Reply:
done!